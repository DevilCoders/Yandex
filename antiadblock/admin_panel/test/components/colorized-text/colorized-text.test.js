import React from 'react';
import {render} from 'enzyme';
import toJson from 'enzyme-to-json';

import ColorizedText from 'app/components/colorized-text/colorized-text';

describe('colorized-text component', () => {
    test('green color', () => {
        const wrapper = render(<ColorizedText color='green' />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('empty color', () => {
        const wrapper = render(<ColorizedText />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
