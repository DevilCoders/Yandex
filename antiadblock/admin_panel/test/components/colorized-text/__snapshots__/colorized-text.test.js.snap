// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colorized-text component empty color 1`] = `
<div
  class="colorized-text colorized-text_color_default"
/>
`;

exports[`colorized-text component green color 1`] = `
<div
  class="colorized-text colorized-text_color_green"
/>
`;
